src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -130,40 +131,37 @@ const BioAndChat = (props) => {
           }
         />
       </TouchableOpacity>
-      <TouchableOpacity onPress={toggleSwitch} style={styles.toggleSwitchContainer}>
-        <ToggleSwitch
-          value={isAnonimity}
-          onValueChange={toggleSwitch}
-          labelLeft={
-            isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
-          }
-          styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
-        />
-      </TouchableOpacity>
+      <ToggleSwitchAnon
+        value={isAnonimity}
+        onValueChange={toggleSwitch}
+        labelLeft={
+          isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
+        }
+      />
     </View>
   );
 };
 
 const styles = StyleSheet.create({
-  bioAndSendChatContainer: (isAnonimity) => ({
-    backgroundColor: isAnonimity ? colors.bondi_blue : colors.blue1,
+  bioAndSendChatContainer: (dynamicColors) => ({
+    backgroundColor: dynamicColors.primary,
     borderRadius: 15,
     paddingHorizontal: 10,
     paddingTop: 10
   }),
   containerBio: {
     marginBottom: 10
   },
-  bioText: (isAnonimity) => ({
-    color: isAnonimity ? colors.greenDark : colors.white,
+  bioText: (dynamicColors) => ({
+    color: dynamicColors.text,
     fontSize: 14,
     fontWeight: '600',
     lineHeight: 22
   }),
   seeMore: {
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are replacing the hardcoded color values with dynamic ones. This is a good practice as it allows for easier theme changes and better consistency across the app. However, ensure that the `dynamicColors` object contains the `primary` and `text` properties, otherwise, you might encounter undefined values.

```diff
- color: isAnonimity ? colors.greenDark : colors.white,
+ color: dynamicColors.text,
```
In line 156, the hardcoded color value has been replaced with a dynamic one. This is a good practice as it maintains consistency across the application and makes it easier to manage color changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->